docs/Usage_example_FDP.ipynb

```diff
@@ -505,17 +384,21 @@
     "        title=[LiteralField(value=record[\"name\"])],\n",
     "        description=[LiteralField(value=record[\"description\"])],\n",
     "        identifier=record[\"id\"],\n",
-    "        is_part_of = [catalog_fdp_id],\n",
+    "        is_part_of=[f\"{fdp_base}/catalog/{catalog_fdp_id}\"],\n",
     "        creator=[record[\"author_id\"]],\n",
     "        release_date=record[\"issued\"],\n",
-    "        publisher=[record[\"publisher_id\"]],\n",
+    "        publisher=[Agent(name=[record[\"publisher_name\"]], identifier=record[\"publisher_id\"])],\n",
     "        theme=record[\"theme\"],\n",
-    "        keyword=[LiteralField(value=x) for x in record[\"keywords\"]]\n",
+    "        keyword=[LiteralField(value=x) for x in record[\"keywords\"]],\n",
+    "        has_version=\"0.1\",\n",
     "    )\n",
     "    dataset_subject = URIRef(f\"http://example.com/dataset_{record['id'][0]}\")\n",
     "    dataset_graph = dataset.to_graph(dataset_subject)\n",
     "    print(dataset_graph.serialize())\n",
-    "    fdp_client.create_and_publish(resource_type=\"dataset\", metadata=dataset_graph)\n"
+    "    dataset_fdp_id = fdpclient.create(type=\"dataset\", data=dataset_graph)\n",
+    "    fdpclient.update(\n",
```

Comment:
RuntimeError                              Traceback (most recent call last)
Cell In[27], line 18
     16 dataset_graph = dataset.to_graph(dataset_subject)
     17 print(dataset_graph.serialize())
---> 18 dataset_fdp_id = fdpclient.create(type="dataset", data=dataset_graph)
     19 fdpclient.update(
     20     type="dataset", subtype="meta/state", id=dataset_fdp_id, data='{"current": "PUBLISHED"}', format="json-ld"
     21 )

File ~/Github/SeMPyRO/venv/lib/python3.9/site-packages/fdpAPIconnector/fdpclient.py:98, in FDPClient.create(self, type, data, format)
     96 except Exception as error:
     97     print(f'Unexpected error when connecting to {self.baseurl}/{type}\n')
---> 98     raise error

File ~/Github/SeMPyRO/venv/lib/python3.9/site-packages/fdpAPIconnector/fdpclient.py:94, in FDPClient.create(self, type, data, format)
     91     if r.status_code >= 300:
     92         print(f'HTTP error: {r.status_code} {r.reason} for {self.baseurl}/{type}',
     93               f'\nResponse message: {r.text}')
---> 94         raise
     95     return self._removePrefix(r.headers['Location'],f'{self.publicurl}/{type}/')
     96 except Exception as error:

RuntimeError: No active exception to reraise

Reply:
Bij mij gaat het niet helemaal lekker